libpod/pod_api.go

```diff
@@ -392,6 +401,54 @@ func (p *Pod) Status() (map[string]ContainerStatus, error) {
 	return status, nil
 }
 
+// Status determines the status of the pod based on the
```

Comment:
I don't think this belongs here... We really need a package for code shared between Varlink and cmd/podman. Maybe rename cmd/podman/batchcontainer to cmd/podman/shared and stick it in there?

Reply:
I can put getPodStatus into batchcontainer for now, then spin up a second pr for the name switch? WDYT @mheon

Reply:
I think it being part of this PR is fine

Reply:
fixed

Reply:
@mheon now that I look at it, should it be moved to pkg/ instead of cmd/podman ?

Reply:
I think cmd/podman/ is still a good place for now.